.env.example

```
@@ -1,7 +1,22 @@
 
 SEPOLIA_RPC=
-SEPOLIA_DEPLOYER_NAME=
+
+BASE_RPC=
+ARBITRUM_RPC=
+GNOSIS_RPC=
+CELO_RPC=
+OPTIMISM_RPC=
+POLYGON_RPC=
+
+PRIVATE_KEY=   # required: private key for deployer / Forge scripts
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)